### PR TITLE
Update quotes for govuk-platform-health

### PR DESCRIPTION
**Pull request by hongoose**

### CODE REVIEW

File: config/alphagov.yml [hunk 1]
```diff
@@ -127,6 +127,30 @@ govuk-platform-health:
...
+    - Ask for more clarity if needed
+    - Feel free to ask for help/questions/etc
+    - Try pairing, as well as not pairing...both are good
+    - Ask questions when you don't know something - it will also give others the confidence to ask their questions
+    - It's OK to not know how something works
+    - Be supportive - knowledge sharing helps the team achieve its goals
+    - Don't rush on to the next thing, pause, help out, deliver work as a team
+    - Work at a considered / sustainable pace 
+    - Take responsibility for your work until it's deployed
+    - Awareness of everyone's preferences on ways of working (more info: https://drive.google.com/drive/folders/1qllr2HfwGg6JL1VCWPCQusMokYYCq_-Z)
```
Comment (thomasleese) [inline config/alphagov.yml]: You need to put this one around quote marks (") because it has a `:` in it.

Comment (thomasleese) [reply config/alphagov.yml]: `- "Awareness of everyone's preferences on ways of working (more info: https://drive.google.com/drive/folders/1qllr2HfwGg6JL1VCWPCQusMokYYCq_-Z)"`

**Review by thomasleese — APPROVED** [review]

File: config/alphagov.yml (hunk 1 above)
Comment (hongoose) [reply config/alphagov.yml]: Fanks Tom

**Comment by kevindew** [comment]
Closed due to internal PR

**Pull request closed without merging**